applications/app/views/newspaperPages.scala.html

```diff
@@ -0,0 +1,26 @@
+@(index: services.IndexPage, nav: PreviousAndNext)(implicit request: RequestHeader)
```

Comment:
Ah yes, thanks. That was left over from the earliest days of this thing's existence. Should have been binned a while back. Will do that now.